app/build.gradle.kts

```diff
@@ -12,12 +12,12 @@ plugins {
 
 android {
     namespace = "com.sebastianvm.mango"
-    compileSdk = 33
+    compileSdk = 34
```

Comment:
Did you not get complains about AGP version?

I had to use `compilePreviewSdk = "UpsideDownCakePrivacySandbox` to get it to work without warnings

Reply:
yea I did so I added the suppression in the gradle.properties to shut it up 😂